lib/order.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'formatter'
+
+class Order
+  include Formatter
+
+  attr_reader :list
+
+  def initialize(time_klass = Time)
+    @time_klass = time_klass
+    @list = Array.new
+  end
+
+  def add(dish, quantity)
+    @list << {dish: dish, quantity: quantity}
+  end
+
+  def total
+    total = 0
+    @list.each do |item|
+      price, quantity = item[:dish].price, item[:quantity]
+      total += price * quantity
+    end
+    return total
+  end
+
+  def receipt
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for receipt is too high. [17.75/15] (http://c2.com/cgi/wiki?AbcMetric)